src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,132 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 1 << 4;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int threshold;
+    private int capacity;
+    private int size;
+
+    public MyHashMap() {
+        this.threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+        this.capacity = DEFAULT_CAPACITY;
+        this.table = (Node<K, V>[]) new Node[capacity];
+    }
 
     @Override
     public void put(K key, V value) {
 
+        putNode(key, value);
     }
 
     @Override
     public V getValue(K key) {
-        return null;
+        return getNodeValue(key);
     }
 
     @Override
     public int getSize() {
-        return 0;
+        return size;
+    }
+
+    private static class Node<K, V> {
```

Comment:
Place the class at the very end.
And also, all public methods come first, then all private ones.